### ISSUE TITLE
chainer usage problem

### ISSUE THREAD

**Issue by mjiansun** [pfnet/chainer#2506]
**When i use chainer to train my data, it can run successfully. But I don't know if this usage is correct ?** 
The following is the accuracy of my training:
`step 43740, accuracy 0.99, cost 0.0277939`
`step 43741, accuracy 0.99, cost 0.0185901`
`step 43742, accuracy 0.97, cost 0.0861139`
`step 43743, accuracy 0.99, cost 0.0202364`
`step 43744, accuracy 0.99, cost 0.0255361`

In my model.py, i define
`def __call__(self, normal_x, sub_x, mini_x, train=False):`
`h1 = self.conv1_1(normal_x)`
`h1 = F.relu(self.bnorm1_1(h1, test=not train))`
`h2 = self.conv2_1(sub_x)`
`h2 = F.relu(self.bnorm2_1(h2, test=not train))`
`h3 = self.conv3_1(mini_x)`
`c = F.concat((h1,h2),axis=1)`
`c = F.concat((c,h3),axis=1)`

In my train.py, I input 3 Variable. **Is this step right for me?**
`xp = cuda.cupy if args.gpu >= 0 else np`
`normal_train = Variable(xp.asarray(normal_img))`
`sub_train = Variable(xp.asarray(subway_img))`
`mini_train = Variable(xp.asarray(mini_img))`
`y_train = Variable(xp.asarray(label_img))`
`output = dmn(normal_train, sub_train,mini_train, train=True)`
`loss = F.softmax_cross_entropy(output, y_train)`

**Comment by unnonouno** [comment]
It looks good. What is your question?
